packages/sanity-cms/package.json

```diff
@@ -20,12 +20,11 @@
     "@session/sanity-types": "workspace:*",
     "@session/testing": "workspace:*",
     "@session/typescript-config": "workspace:*",
-    "@types/react": "^18.3.1",
-    "@types/react-dom": "^18.3.0",
+    "@types/react": "npm:types-react@19.0.0-rc.1",
+    "@types/react-dom": "npm:types-react-dom@19.0.0-rc.1",
     "localtunnel": "^2.0.2",
-    "next": "14.2.12",
-    "react": "18.3.1",
-    "tailwind": "^4.0.0"
```

Comment:
Do we not still need tailwind since we have a tailwind.config.ts file in this package?

Reply:
Oh actually unless you removed it since it is a part of session/ui?

Reply:
actually its the wrong tailwind (tailwind is tailwindcss), and yeah we dont need it.